### PR TITLE
clean files after e2e test

### PR DESCRIPTION
**Pull request by edisonxiang**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
To clean up files which are produced by e2e test.

### CODE REVIEW

File: tests/e2e/scripts/cleanup.sh [hunk 1]
```diff
@@ -40,5 +40,28 @@ kill_edgecontroller() {
     fi
 }
 
+cleanup_files(){
+    echo $PWD
+    cd $PWD
+    cd ../../
```
Comment (m1093782566) [inline tests/e2e/scripts/cleanup.sh]: Probably we can define the gopath and then find the proper files to delete base on gopath.

Comment (edisonxiang) [reply tests/e2e/scripts/cleanup.sh]: Thanks.

**Comment by sids-b** [comment]
As far as I know, travis will spinup a new container for running each job, and when the job completes, the container will be destroyed. Do we need cleanup script here ??

**Comment by m1093782566** [comment]
I think this PR make sense for non-CI test environment.

**Comment by pavan187** [comment]
> As far as I know, travis will spinup a new container for running each job, and when the job completes, the container will be destroyed. Do we need cleanup script here ??

Yes, it needs a cleanup, when you run locally framework is generating lot of files, its better if we cleanup those things.

File: tests/e2e/scripts/cleanup.sh [hunk 2]
```diff
@@ -40,5 +40,28 @@ kill_edgecontroller() {
...
+
+    sudo rm -rf cloud/edgecontroller/edgecontroller
+    sudo rm -rf cloud/edgecontroller/edgecontroller.log
+    sudo rm -rf cloud/edgecontroller/tmp/
+    sudo rm -rf edge/edge.db
+    sudo rm -rf edge/edge_core
+    sudo rm -rf edge/edge_core.log
```
Comment (pavan187) [inline tests/e2e/scripts/cleanup.sh]: @edisonxiang , do not cleanup module logs, as we may need them for issue debug in case if any failures.

**Comment by edisonxiang** [comment]
Yep, we are running in local env.

**Comment by edisonxiang** [comment]
Done, PTAL, Thanks:)

**Pull request merged**